Kinvey/Kinvey/DataStore.swift

```diff
@@ -103,7 +105,13 @@ open class DataStore<T: Persistable> where T: NSObject {
      - Returns: An instance of `DataStore` which can be a new instance or a cached instance if you are passing a `tag` parameter.
      */
     @available(*, deprecated: 3.0.22, message: "Please use `collection()` instead")
-    open class func getInstance(_ type: StoreType = .cache, deltaSet: Bool? = nil, client: Client = sharedClient, tag: String = defaultTag) -> DataStore {
+    open class func getInstance(
```

Comment:
Needs a reference doc for the new `validationStrategy` parameter.

Reply:
done